chromium_src/components/content_settings/core/browser/content_settings_registry.cc

```diff
@@ -14,7 +14,8 @@ void ContentSettingsRegistry::BraveInit() {
            WebsiteSettingsRegistry::DESKTOP |
                WebsiteSettingsRegistry::PLATFORM_ANDROID,
            ContentSettingsInfo::INHERIT_IN_INCOGNITO,
-           ContentSettingsInfo::PERSISTENT);
+           ContentSettingsInfo::PERSISTENT,
+           ContentSettingsInfo::EXCEPTIONS_ON_SECURE_AND_INSECURE_ORIGINS);
```

Comment:
seems like something that maybe needs security review?

Reply:
cc @diracdeltas

Reply:
diff --git a/components/content_settings/core/browser/content_settings_info.h b/components/content_setti
ngs/core/browser/content_settings_info.h
index a7f691b39e57..834913d7ecc9 100644
--- a/components/content_settings/core/browser/content_settings_info.h
+++ b/components/content_settings/core/browser/content_settings_info.h
@@ -39,12 +39,22 @@ class ContentSettingsInfo {
     EPHEMERAL,
   };

+  enum OriginRestriction {
+    // This flag indicates content types that only allow exceptions to be set
+    // on secure origins.
+    EXCEPTIONS_ON_SECURE_ORIGINS_ONLY,
+    // This flag indicates content types that allow exceptions to be set on
+    // secure and insecure origins.
+    EXCEPTIONS_ON_SECURE_AND_INSECURE_ORIGINS,
+  };
+

Reply:
from https://chromium.googlesource.com/chromium/src.git/+/f2027f0a91ac0128f9a61c472ca7e518898a8729%5E%21/

Reply:
Posted here to track the security review it, also cross posted to the internal security issues:
https://github.com/brave/brave-browser/issues/2620